src/System.Text.Json/tests/System.Text.Json.Tests.csproj

```diff
@@ -40,7 +40,10 @@
     <Compile Include="Serialization\CustomConverterTests.BadConverters.cs" />
     <Compile Include="Serialization\CustomConverterTests.cs" />
     <Compile Include="Serialization\CustomConverterTests.DerivedTypes.cs" />
-    <Compile Include="Serialization\CustomConverterTests.Dictionary.cs" />
+    <Compile Include="Serialization\CustomConverterTests.DictionaryConverterForIDictionary.cs" />
```

Comment:
nit: Be consistent in file names.

`CustomConverterTests.IDictionaryConverter`?

It looks like the other converter files have `Converter` as a suffix.

Reply:
Per above comment -- this was an existing test -- changing name to ContravariantDictionaryConverter.cs